content/en/logs/explorer/analytics/_index.md

```diff
@@ -99,10 +102,20 @@ Apply a formula on one or multiple queries by clicking on the `+ Add` button nex
 
 To apply formulas with multiple queries, all queries must be grouped by the same facet. In the example above, both queries are grouped by `Webstore Store Name`.
 
-You can apply a function to a formula by clicking on the `Σ` icon. Here is an example of how to apply a [Timeshift function][10] on the proportion of error logs in all logs to compare current data with data from one week before:
+You can apply a function to a formula by clicking on the `Σ` icon. Here is an example of how to apply a [Timeshift function][12] on the proportion of error logs in all logs to compare current data with data from one week before:
 
 {{< img src="logs/explorer/group/timeshift_function_logs.jpg" alt="The query editor showing a formula with the week before timeshift function applied to it" style="width:100%;" >}}
 
+## Filter logs based on Reference Tables
+
+<div class="alert alert-warning">Reference Tables and the feature to filter logs based on Reference Tables are in beta. There are no billing implications for defining and querying Reference Tables. There is a limit of 100 Reference Tables per account and a limit of 1000 rows in a Reference Table.</div>
```

Comment:
There may be costs to defining a reference table in the future so let's leave that part out.

Reply:
```suggestion
<div class="alert alert-warning">Reference Tables and the feature to filter logs based on Reference Tables are in beta. There is a limit of 100 Reference Tables per account and a limit of 1000 rows in a Reference Table.</div>
```